test/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/DeclarationXml_DocumentationAndReferencesSpec.scala

```diff
@@ -68,7 +114,7 @@ class DeclarationXml_DocumentationAndReferencesViewModelSpec extends WordSpec wi
     "render a single item level additional information element" in {
       val additionalInformation = AdditionalInformation(Some("foo"), Some("bar"))
       val documentation = DocumentationAndReferencesViewModel().copy(itemAdditionalInformation = Seq(additionalInformation))
-      val declaration = Declaration().copy(documentationAndReferences = documentation)
+      val declaration = Declaration().copy(documentationAndReferences = documentation.toDocumentationAndReferences)
```

Comment:
do we need to involve the viewmodel in this test?  it should just be testing the rendering of xml from the domain model?

Reply:
Ok, I've replaced it with the original DocumentationAndReferences domain class.